### PR TITLE
Andarist/fix production saving crash

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

**Review by mattpocock — CHANGES_REQUESTED** [review]
http://localhost:3000/viz/398fc62e-1820-4979-a2e0-df7cae7f5b89

Machines on these links now don't load.

File: src/App.tsx [hunk 1]
```diff
@@ -34,42 +28,7 @@ function App(props: AppProps) {
       : undefined;
   });
 
-  const router = useRouter();
-
-  const routerReplace = useCallback((url: string) => {
```
Comment (davidkpiano) [inline src/App.tsx:39]: For those wondering, this has been moved to here: https://github.com/statelyai/xstate-viz/pull/245/files#diff-88a36bc7a53bfa58c6f451464798d631d785160dfacfdaf9479be7a0b1d7e5e5R34-R65

**Review by davidkpiano — APPROVED** [review]
Approved pending CI checks

**Review by farskid — APPROVED** [review]
LGTM but CI seems to have failed

**Comment by Andarist** [comment]
@mattpocock good catch, fixed 🤞 

@farskid there were some type errors that I've missed - they are fixed now

**Review by mattpocock — APPROVED** [review]

**Comment by mattpocock** [comment]
@Andarist LGTM!

**Pull request merged**